### PR TITLE
Wait scatter region finishing

### PR DESCRIPTION
**Pull request by zhexuany**
test in local machine.

### CODE REVIEW

File: tikv-client/src/main/java/com/pingcap/tikv/PDClient.java [hunk 1]
```diff
@@ -42,33 +44,75 @@
...
     implements ReadOnlyPDClient {
-
+  private final Logger logger = LoggerFactory.getLogger(PDClient.class);
   private RequestHeader header;
   private TsoRequest tsoReq;
   private volatile LeaderWrapper leaderWrapper;
   private ScheduledExecutorService service;
   private List<URI> pdAddrs;
 
+  private GetOperatorResponse getOperator(long regionId) {
```
Comment (marsishandsome) [inline tikv-client/src/main/java/com/pingcap/tikv/PDClient.java]: pls add some notes to let others know what this function does

Comment (marsishandsome) [reply tikv-client/src/main/java/com/pingcap/tikv/PDClient.java]: cause `METHOD_GET_OPERATOR` does not explain itself.

File: tikv-client/src/main/java/com/pingcap/tikv/TiSession.java [hunk 2]
```diff
@@ -182,11 +183,16 @@ public void splitRegionAndScatter(List<byte[]> splitKeys) {
     logger.info(String.format("split key's size is %d", splitKeys.size()));
     List<Key> rawKeys =
         splitKeys.parallelStream().map(RowKey::toRawKey).collect(Collectors.toList());
-    rawKeys.forEach(this::splitRegionAndScatter);
+    TLongArrayList regionIds = new TLongArrayList();
+    rawKeys.forEach(key -> splitRegionAndScatter(key, regionIds));
+    // TODO: adding a conf let user to decide wait or not.
```
Comment (marsishandsome) [inline tikv-client/src/main/java/com/pingcap/tikv/TiSession.java]: i recommend to add it now, it's simple.

File: tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java [hunk 3]
```diff
@@ -47,6 +47,8 @@
   int SPLIT_REGION_BACKOFF = 20 * seconds;
   int BATCH_PREWRITE_BACKOFF = 20 * seconds;
   int BATCH_COMMIT_BACKOFF = 3 * seconds;
+  int SCATTER_REGION_FINISH = 120 * seconds;
```
Comment (marsishandsome) [inline tikv-client/src/main/java/com/pingcap/tikv/util/BackOffer.java]: unused

File: tikv-client/src/main/java/com/pingcap/tikv/PDClient.java (hunk 1 above)
Comment (zhexuany) [reply tikv-client/src/main/java/com/pingcap/tikv/PDClient.java]: `METHOD_GET_OPERATOR` means get pd's operator associated with specific region.

Comment (marsishandsome) [reply tikv-client/src/main/java/com/pingcap/tikv/PDClient.java]: cool

File: tikv-client/src/main/java/com/pingcap/tikv/TiSession.java (hunk 2 above)
Comment (zhexuany) [reply tikv-client/src/main/java/com/pingcap/tikv/TiSession.java]: My plan is to do it with `enable region pre-split`. What do you think?

**Pull request merged**